src/wrover_status_light/src/DemoServer.cpp

```diff
@@ -0,0 +1,20 @@
+#include "ros/ros.h"
+#include "ros/init.h"
+#include <wrover_status_light/DoubleService.h>
+
+bool addTwoIntsCallback(wrover_status_light::DoubleServiceRequest& req,
```

Comment:
Why is this the `addTwoIntsCallback`?  Is there a better name?

Reply:
Typo on my part. Resolved in [2f9628e](https://github.com/WisconsinRobotics/WRoverPlayground/pull/9/commits/2f9628ede9fad3226413a596c6efa382c0c4ecd7).